pkg/controller/cronjob/cronjob_controllerv2_test.go

```diff
@@ -51,11 +51,9 @@ var (
 	// schedule is hourly on the hour
 	onTheHour = "0 * * * ?"
 
-	A = batchv1.AllowConcurrent
-	f = batchv1.ForbidConcurrent
-	R = batchv1.ReplaceConcurrent
-	T = true
-	F = false
+	AllowConcurrent   = batchv1.AllowConcurrent
```

Comment:
prefer to get rid of these constants as well

Reply:
done